SU2_CFD/src/output/CFlowIncOutput.cpp

```diff
@@ -892,7 +905,7 @@ void CFlowIncOutput::LoadVolumeData(CConfig *config, CGeometry *geometry, CSolve
   if(config->GetKind_TimeIntScheme_Flow()==EULER_IMPLICIT){
     SetVolumeOutputValue("TIMESTEP", iPoint, Node_Flow->GetDelta_Time(iPoint));
   }
-    
+
```

Comment:
please do not run TrimTrailingWhitespaces on those large files, it just inflates the diff and is done best in seperate PR's or at the very end. (And yes I am aware of the Hide trailing whitespaces option on github ;) ) You might get merge conflicts just because of that